javascript/javascript3/week1/Find-a-cool-api.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Find a cool api</title>
+  </head>
+  <body>
+    Food and drinks -https://developer.edamam.com/edamam-docs-nutrition-api
+    The above api consists of reciepies within the objects.That object has an array.
```

Comment:
Here instead of writing into text better  to log those json data.usng fetch.

Reply:
ok jyothi